### PR TITLE
Add an aggregator and mapper to extend comparison mappers to tick, lat, lon, etc.

### PR DESCRIPTION
**Pull request by ftence**
This implementation is around 1.4% slower than the original one.

### CODE REVIEW

File: warp10/src/main/java/io/warp10/script/WarpScriptLib.java [hunk 1]
```diff
@@ -1059,12 +1060,42 @@
     addNamedWarpScriptFunction(new NULL(NULL));
     addNamedWarpScriptFunction(new ISNULL("ISNULL"));
     addNamedWarpScriptFunction(new MapperReplace.Builder("mapper.replace"));
-    addNamedWarpScriptFunction(new MAPPERGT("mapper.gt"));
-    addNamedWarpScriptFunction(new MAPPERGE("mapper.ge"));
-    addNamedWarpScriptFunction(new MAPPEREQ("mapper.eq"));
-    addNamedWarpScriptFunction(new MAPPERNE("mapper.ne"));
-    addNamedWarpScriptFunction(new MAPPERLE("mapper.le"));
-    addNamedWarpScriptFunction(new MAPPERLT("mapper.lt"));
+    addNamedWarpScriptFunction(new MapperCompareTo("mapper.gt", CompareTo.Compared.VALUE, CompareTo.Comparison.GT));
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/script/WarpScriptLib.java]: Could you use constants as per PR #538

**Comment by hbs** [comment]
Did you identify the cause of the lesser performance?

File: warp10/src/main/java/io/warp10/script/WarpScriptLib.java (hunk 1 above)
Comment (ftence) [reply warp10/src/main/java/io/warp10/script/WarpScriptLib.java]: Done.

**Comment by ftence** [comment]
No, i did not used a profiler to check that but I expected such decrease in performance because of the use of `signum`, `compare`, `==` and `^` instead of a single comparison operator.

I think that the flexibility this PR adds outweighs the small performance loss. If performance is seen as critical, the current code can be optimized but I fear it would be far less readable.

**Pull request merged**